web/html/src/manager/minion/ansible/ansible-control-node.tsx

```diff
@@ -221,7 +221,8 @@ export class AnsibleControlNode extends React.Component<PropsType, StateType> {
                 <NewAnsiblePath
                   title={t("Add an Inventory file")}
                   pathType="inventory"
-                  newInventoryPath={this.state.newInventoryPath}
+                  // TODO: @parlt91: Is this bugfix correct?
```

Comment:
@parlt91 I added missing types to `NewAnsiblePath` and found that this prop was not actually connected to anything. I'm _guessing_ it should be the change I made, but I'm not sure and don't know how to test, can you please confirm?